pkg/controller/bastion/actuator_reconcile_test.go

```diff
@@ -119,6 +119,28 @@ var _ = Describe("Bastion Host Reconcile", func() {
 			HaveField("Data", HaveLen(1)),
 		))
 
+		By("ensuring bastion network policy is created with correct spec")
+		Expect(err).ShouldNot(HaveOccurred())
```

Comment:
What `err` is here checked for?

Reply:
It was misplaced there instead of

```
By("ensuring bastion host is created with correct spec")
bastionHostName, err := generateBastionHostResourceName(cluster.ObjectMeta.Name, bastion)
Expect(err).ShouldNot(HaveOccurred())
```